package.json

```diff
@@ -4,7 +4,7 @@
   "keywords": [
     "appium"
   ],
-  "version": "7.3.1",
+  "version": "7.3.3",
```

Comment:
should we make this 8.0.0-beta? or something? on this branch?

Reply:
good idea. This PR only has 'merge master' diff, so it should be another PR like such bumping version :)